common/cgroup/cgroup.go

```diff
@@ -295,7 +346,7 @@ func CreateCgroups(root string, subcgroup string, serviceNames []string) error {
 			fixCpusetKnobs(cPath)
 		}
 
-		// 3c. Create cgroup directories and mount the files we need over
+		// 1b. Create cgroup directories and mount the files we need over
```

Comment:
not sure about keeping the numbering scheme ("1b"...). It's annoying to have to update them!